[7.0][IMP][account_financial_report_webkit] Added new field 'new_page_after_each_account'

BT-ojossen opened the pull request:
added new field 'new_page_after_each_account'

Commit:
added new field 'new_page_after_each_account'

Commit:
removed unwanted commit

Commit:
added translations

Commit:
added new field 'new_page_after_each_account'

BT-ojossen committed:
Merge pull request #2 from BT-ojossen/ojossen_new_page_after_each_account

added new field 'new_page_after_each_account'

BT-ojossen committed:
Merge pull request #1 from BT-ojossen/7.0

added new field 'new_page_after_each_account'

pedrobaeza commented:
Please check Travis errors.

bt-admin committed:
PEP8 adjustments

bt-admin committed:
PEP8 adjustments

bt-admin committed:
PEP8 adjustments

gurneyalex commented:
nice and useful module. @BT-ojossen we are missing some CLA, though. 

:+1: once CLA issue is resolved.

Commit:
added context to browse

added context to browse of account.account in order to get the correct
translation

Commit:
added context to browse

added context to browse of account.account in order to get the correct
translation

Commit:
added context to browse

added context to browse of account.account in order to get the correct
translation

Commit:
added context to browse

added context to browse of account.account in order to get the correct
translation

Commit:
added context to browse

added context to browse of account.account in order to get the correct
translation

Commit:
added context to browse

added context to browse of account.account in order to get the correct
translation

BT-ojossen committed:
Merge pull request #9 from BT-ojossen/7.0_ojossen_add_ctx_to_browse

7.0 ojossen add ctx to browse

Commit:
Merge branch '7.0' of https://github.com/OCA/account-financial-reporting into OCA-7.0

Conflicts:
	account_financial_report_webkit/report/common_balance_reports.py
	account_financial_report_webkit/report/common_partner_balance_reports.py
	account_financial_report_webkit/report/general_ledger.py
	account_financial_report_webkit/report/open_invoices.py
	account_financial_report_webkit/report/partners_ledger.py

Commit:
Merge branch 'OCA-7.0' into 7.0

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

BT-ojossen committed:
flake8 bugfixes

Commit:
Rename create_single_pdf to create_single_pdf_old 

To make compatible with bt_report_webkit

Commit:
Merge pull request #11 from brain-tec/BT-csanchez-create_single_pdf

Rename create_single_pdf to create_single_pdf_old

BT-ojossen committed:
set cumul_balance_curr in report

Commit:
Merge branch '7.0' of git://github.com/OCA/account-financial-reporting into 7.0

Conflicts:
	account_financial_report_webkit/i18n/fr.po

bt-admin committed:
Merge pull request #77 from OCA/7.0

Syncing from upstream OCA/account-financial-reporting (7.0)

feketemihai commented:
@BT-ojossen I will close one since no activity for a long time. Please reopen it if still need it.
